app/views/teaching_events/about_git_events.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= render Campaign::HeaderComponent.new(
+  title: ["Get Into Teaching", "events"],
+  colour: "yellow",
+  image: "media/images/content/hero-images/0025.jpg"
+) %>
+
+<div class="subject-specific">
+  <article>
+    <section class="statement">
+      <span class="pink xlarge bold">What to expect</span>
+    </section>
+
+    <section>
+      <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+
+      <p>They're a series of relaxed and informative events giving you the opportunity to get expert advice and talk to teachers and local training providers.</p>
+
+      <h2 class="heading-m">At the event</h2>
+
+      <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
+
+      <p>You can discover:</p>
+
+      <ul>
+        <li>if teaching is right for you</li>
+        <li>how to apply for teacher training</li>
+        <li>what funding is available</li>
+      </ul>
+
+      <h2 class="heading-m">Reserve your place</h2>
+
+      <p>You'll need to register to attend a Get Into Teaching event. Places are limited, so the sooner you sign up, the better.</p>
+    </section>
+
+    <section class="grey-bg fullwidth">
+      <%= render SubjectSpecific::TextBlockWithOneImageComponent.new(
+        heading: "Events in your region",
+        colour: "blue",
+        image_path: "media/images/content/homepage/here-to-help.jpg",
+        image_alt: "Child pointing something out to a teacher in the classroom.",
+      ) %>
+    </section>
+
+    <section class="fullwidth grey-bg no-padding no-margin">
+      <section class="container no-margin">
+        <section class="category__cards">
+          <%= tag.nav(class: "category__nav-cards") do %>
+            <ul class="single-column">
+              <%= render(Categories::CardComponent.with_collection(categorise_events(@git_events))) %>
+            </ul>
+          <% end %>
+        </section>
+      </section>
+    </section>
+
+    <section class="fullwidth grey-bg no-margin">
+      <section class="container no-margin bottom-padding">
+        <div>
+          <h2 class="heading-s">Can't find what you're looking for?</h2>
+          <p><%= link_to("View all events", events_path, class: "link--bold") %></p>
```

Comment:
@gazcarr @GDW7 I wonder if we need to make it clearer that this link goes to other types of events as well?

Reply:
I think this works ok personally, we could add more here though... when all of the events are gone the page will be left with this so maybe @GDW7 bear that in mind?

Reply:
Let's deal with when all the events are gone separately (I'll raise a ticket now so we don't forget!) - this can be done post live in a more elegant way.

I'd suggest just considering this text in terms of how it sounds now and up to the 6 Dec (when the last of these events takes place) if that makes sense?